.storybook/preview.tsx

```diff
@@ -1,6 +1,8 @@
 import type { Preview } from '@storybook/react'
 
-import { SeamProvider } from 'index.js'
+import '../lib/styles/index.css'
```

Comment:
We will need to document how to import the styles with our steps to using the lib.

Reply:
Added simple import for now in  https://github.com/seamapi/react/pull/47/commits/da4137d6ad849297e4dcdeb9a8eacb6501e49492

Reply:
We should definitely circle back, and update docs once everything is more production-ready.